adapters/YahooAdapter/YahooAdapter/GADMAdapterYahooConstants.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+/// Yahoo mediation network adapter version.
+static NSString *const _Nonnull GADMAdapterYahooVersion = @"1.14.1.1";
```

Comment:
Please update to version "1.4.0.1"  - e.g. version 1 of the adapter for Yahoo Mobile SDK v1.4.0

Reply:
Updated.